### PR TITLE
27078 - updated the es kubectl installation docs

### PR DESCRIPTION
**Pull request by SivaTharun**
- PR to address #27078 
- removed the old kubectl installtion markdown file
- followed the hierarchy for en and introduced the markdown files
- added markdown files for macos,Linux and windows operating systems

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info https://github.com/kubernetes/website/pull/29113#issuecomment-886190766

File: content/es/docs/tasks/tools/included/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Herramientas incluidas"
+description: "Fragmentos que se incluirán en la principal kubectl-installs-*.md paginas."
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/_index.md:3]:
```suggestion
description: "Fragmentos que se incluirán en las paginas principaldes de kubectl-installs-*."
```

File: content/es/docs/tasks/tools/included/kubectl-convert-overview.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "descripción general de kubectl-convert"
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/kubectl-convert-overview.md:2]:
```suggestion
title: "Descripción general de kubectl-convert"
```

File: content/es/docs/tasks/tools/included/kubectl-convert-overview.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+description: >-
+  Un complemento de kubectl que le permite convertir manifiestos de una versión
+  de una API de Kubernetes a una versión diferente.
+headless: true
+---
+
+Un complemento para la herramienta de línea de comandos de Kubernetes `kubectl`, que le permite convertir manifiestos entre diferentes API versiones. Esto puede ser particularmente útil para migrar manifiestos a una versión de api no obsoleta con la versión más reciente de Kubernetes.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/kubectl-convert-overview.md:9]:
```suggestion
Un complemento para la herramienta de línea de comandos de Kubernetes `kubectl`, que le permite convertir manifiestos entre diferentes versiones de API. Esto puede ser particularmente útil para migrar manifiestos a una versión de API no obsoleta con la versión más reciente de Kubernetes.
```

File: content/es/docs/tasks/tools/included/kubectl-convert-overview.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+Para obtener más información, visite [migrar a apis no obsoletas](/docs/reference/using-api/deprecation-guide/#migrate-to-non-deprecated-apis)
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/kubectl-convert-overview.md:10]:
```suggestion
Para obtener más información, visite [migrar a APIs no obsoletas](/docs/reference/using-api/deprecation-guide/#migrate-to-non-deprecated-apis)
```

File: content/es/docs/tasks/tools/included/kubectl-whats-next.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: "¿Que sigue?"
+description: "¿Qué sigue después de instalar kubectl."
+headless: true
+---
+
+* [Instalar Minikube](https://minikube.sigs.k8s.io/docs/start/)
+* Consulte las [guías de introducción](/docs/setup/) para obtener más información sobre la creación de clústeres.
+* [Aprenda a iniciar y exponer su aplicación.](/docs/tasks/access-application-cluster/service-access-application-cluster/)
+* Si necesita acceso a un clúster que no creó, consulte la
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/kubectl-whats-next.md:10]:
```suggestion
* Si necesita acceso a un clúster que no creó, consulte la guia de
```

File: content/es/docs/tasks/tools/included/kubectl-whats-next.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  [Compartir documento de acceso al clúster](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/kubectl-whats-next.md:11]:
```suggestion
  [Compartir el acceso al clúster](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+title: "Autocompletar bash en Linux"
+description: "Alguna configuración opcional para la finalización automática de bash en Linux."
+headless: true
+---
+
+### Introducción
+
+El script de finalización de kubectl para Bash se puede generar con el comando `kubectl completion bash`. Obtener el script de finalización en su shell habilita el autocompletado de kubectl.
+
+However, the completion script depends on [**bash-completion**](https://github.com/scop/bash-completion), which means that you have to install this software first (you can test if you have bash-completion already installed by running `type _init_completion`).
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:11]: This paragraph wasn't translated

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+### Instalar bash-complete
+
+finalización de bash es proporcionado por muchos administradores de paquetes (ver [aquí](https://github.com/scop/bash-completion#installation)). Puedes instalarlo con `apt-get install bash-completion` o `yum install bash-completion`, etc.
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:15]:
```suggestion
La finalización de bash es proporcionado por muchos administradores de paquetes (ver [aquí](https://github.com/scop/bash-completion#installation)). Puedes instalarlo con `apt-get install bash-completion` o `yum install bash-completion`, etc.
```

File: content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+Los comandos anteriores crean `/usr/share/bash-completion/bash_completion`, que es el guión principal de bash-complete. Dependiendo de su administrador de paquetes, debe obtener manualmente este archivo en su `~/.bashrc` expediente.
+
+Para averiguarlo, recargue su caparazón y ejecute `type _init_completion`. Si el comando tiene éxito, ya está configurado; de lo contrario, agregue lo siguiente a su `~/.bashrc` expediente:
```
Comment (electrocucaracha) [inline content/es/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md:19]:
```suggestion
Para averiguarlo, recargue su shell y ejecute `type _init_completion`. Si el comando tiene éxito, ya está configurado; de lo contrario, agregue lo siguiente a su archivo `~/.bashrc`:
```

**Review by electrocucaracha — COMMENTED** [review]
These are some initial reviews, please sign the CLA document and address those comments.

**Comment by raelga** [comment]
@SivaTharun did you need any help with the CLA? Feel free to ping me over slack at any time.

/check-cla

**Comment by raelga** [comment]
Please @SivaTharun let me know if we can help you with the CLA :)

**Comment by raelga** [comment]
Hello @SivaTharun, thanks for your PR but without the CLA, it cannot be merged. Please let me know if you can to try it again in the future, I'll be more than happy to help you get the CLA.

/close

**Pull request closed without merging**